### ISSUE TITLE
L2_normalization layer instead of sigmoid  (in Contrastive loss)

### ISSUE THREAD

**Issue by c100rczyk** [c100rczyk/Team_project_PWR#24]
https://stackoverflow.com/questions/59596162/how-to-apply-l2-normalization-to-a-layer-in-keras

**Comment by c100rczyk** [comment]
With BatchNormalization + Sigmoid  , layers the accuracy(top1) was 82%  and top5 accuracy was 97.7%

**Comment by c100rczyk** [comment]
Deleted sigmoid. Added L2 normalization.  Model have difficulty to learn. (Earlier , befor change , it was same problem sometimes). But after get Model with Transfer Learning , model learns good , and everything is ok
